unskript-ctl/db_utils.py

```diff
@@ -173,6 +173,45 @@ def delete_pss_record(record_name: str, document_name: str) -> bool:
     return True
 
 
+def get_checks_by_uuid(check_uuid_list: list):
```

Comment:
This is inefficient, we should query the DB to get this. Change the schema of the DB if required.

Reply:
Dont get the comment. ZoDB does not have a search engine, we store list of dictionaries (code snippet list). The code queries the ZoDB and fetch the checks that match the check UUIDs that were sent as a list.